Mathlib/Analysis/NormedSpace/PiLp.lean

```diff
@@ -76,6 +76,11 @@ abbrev PiLp (p : ℝ≥0∞) {ι : Type*} (α : ι → Type*) : Type _ :=
   WithLp p (∀ i : ι, α i)
 #align pi_Lp PiLp
 
+/-The following should not be a FunLike instance because then the coercion ⇑ would get
+unfolded to FunLike.coe instead of WithLp.equiv.-/
```

Comment:
```suggestion
/-The following should not be a `FunLike` instance because then the coercion `⇑` would get
unfolded to `FunLike.coe` instead of `WithLp.equiv`.-/
```